packages/test-utils/sdks/operationid-quirks/src/index.ts

```diff
@@ -106,6 +106,10 @@ class SDK {
   }
 }
 
+/**
+ * Quirky `operationID` cases
+ *
+ */
 const createSDK = (() => { return new SDK(); })();
```

Comment:
Moving these docblocks from the `SDK` class to this export caused them to get surfaced:

![screen_shot_2023-10-25_at_10 02 38_am](https://github.com/readmeio/api/assets/33762/20111cd5-fdd3-4df5-bc27-48550d7cd31e)